mkdocs.yml

```diff
@@ -13,12 +13,27 @@ plugins:
   - redirects:
       redirect_maps:
         "training_resources/git/intro-to-git.md": "training_resources/git/introduction-to-git.md"
+        "about.md": "site_info/about.md"
+        "our_RAP_service/thin-slice-strategy.md": "implementing_RAP/thin-slice-strategy.md"
+        "glossary.md": "site_info/glossary.md"
```

Comment:
it won't let me comment on the glossary page:
But I think unhide the navigation on that page